service/controller/v18/version_bundle.go

```diff
@@ -7,6 +7,11 @@ import (
 func VersionBundle() versionbundle.Bundle {
 	return versionbundle.Bundle{
 		Changelogs: []versionbundle.Changelog{
+			{
+				Component:   "aws-operator",
+				Description: "Fixed a bug with removing CR finalizers before all Cloud Formation stacks are deleted.",
+				Kind:        versionbundle.KindFixed,
+			},
```

Comment:
I'm still waiting for the review here. Doesn't it make sense to add it to the changelog?

cc @teemow